Gemfile

```
@@ -29,6 +29,8 @@ gem 'doorkeeper' # OAuth 2 provider functionality to Ruby on Rails
 gem 'rails-i18n' # Translations for Rails
 # gem 'devise-i18n' # Translations for Devise
 
+gem 'grover'
```

Follow other existing gem definition, please add the short description after the name. 

```suggestion
gem 'grover'  # A Ruby gem to transform HTML into PDFs, PNGs or JPEGs
```